apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts

```diff
@@ -78,4 +87,19 @@ export class NotificationsController {
     this.logger.info('Message queued', { messageId: id, ...message })
     return { id }
   }
+
+  @Get('/test')
+  async test(): Promise<any> {
+    return { date: new Date().toISOString() }
```

Comment:
I think you need to `await` here, might be unrelated to the [pipeline failure issue](https://github.com/island-is/island.is/actions/runs/4065092505/jobs/6999334430), but maybe you can give it a shot?